synapse/storage/events.py

```diff
@@ -905,3 +905,59 @@ def _has_been_redacted_txn(self, txn, event):
         txn.execute(sql, (event.event_id,))
         result = txn.fetchone()
         return result[0] if result else None
+
+    @defer.inlineCallbacks
+    def count_daily_messages(self):
```

Comment:
Docstrings might be nice. Possibly worth calling out that it returns None if it's been too long since it last reported?